docs/pages/access-controls/reference.mdx

```diff
@@ -89,14 +89,20 @@ spec:
     max_sessions: 10
     # permit_x11_forwarding allows users to use X11 forwarding with openssh clients and servers through the proxy
     permit_x11_forwarding: true
-    # Specify whether or not to record the user's desktop sessions.
-    # Desktop session recording is enabled if one or more of the user's
-    # roles has enabled recording. Defaults to true if unspecified.
-    # Desktop sessions will never be recorded if auth_service.session_recording
-    # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
-    # resource has set 'mode: off'.
+    # Session recording options.
     record_sessions:
+      # Specify whether or not to record the user's desktop sessions.
+      # Desktop session recording is enabled if one or more of the user's
+      # roles has enabled recording. Defaults to true if unspecified.
+      # Desktop sessions will never be recorded if auth_service.session_recording
+      # is set to 'off' in teleport.yaml or if the cluster's session_recording_config
+      # resource has set 'mode: off'.
       desktop: true
+      # Optional: Session recording mode default value used when sessions kind
+      # doesn't have one set.
+      default: best_effort|strict
+      # Optional: Session recording mode for SSH sessions (server access).
```

Comment:
```suggestion
      # Optional: Session recording mode for SSH sessions (Teleport Server Access).
```

This is optional because the default is set by `default`?

Reply:
Yes.